parameter/DefaultElementLibrary.h

```diff
@@ -46,7 +47,13 @@ class CDefaultElementLibrary: public CElementLibrary
 {
 public:
 
-    explicit CDefaultElementLibrary(bool bEnableDefaultMechanism = true);
+    /**
+     * Class constructor
+     *
+     * @param[in] logger the logger provided by the client
+     * @param[in] bEnableDefaultMechanism boolean allowing to activate VirtualSubsystem use
+     */
+    explicit CDefaultElementLibrary(core::log::ILogger& logger, bool bEnableDefaultMechanism = true);
```

Comment:
No it is still needed
~~~
struct I { I(int a, int b=0) {} };
struct E { explicit E(int a, int b=0) {} };
struct F { static void i(I) {}
               static void e(E) {} };
F::i(1); // OK
F::e(1); // Error: no viable conversion from 'int' to 'E'
~~~
Note: I wrapped i(I) and e(E) in a class F because cling does not allow function definition out of a class.